.circleci/config.yml

```diff
@@ -133,6 +148,76 @@ jobs:
       - run:
           name: Run pylint
           command: ./tools/pylint.sh
+  setup-test-reporter:
+    docker:
+      - image: cimg/base:stable
+    resource_class: small
+    steps:
+      - attach_workspace:
+          at: .
+      - run:
+          name: Install CodeClimate Test Reporter
+          command: |
+            curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+            chmod +x ./cc-test-reporter
+      - run:
+          name: Notify CodeClimate of a pending report
+          command: ./cc-test-reporter before-build
+      - persist_to_workspace:
+          root: .
+          paths:
+            - cc-test-reporter
+  test:
+    docker:
+      - image: cimg/python:3.9.16
+      - image: cimg/postgres:14.1
+        environment:
+          POSTGRES_USER: integreat
+          POSTGRES_DB: integreat-compass
+          POSTGRES_PASSWORD: password
+    resource_class: small
+    parallelism: 16
```

Comment:
Indeed, since 14 of the 16 containers do nothing than to spin up and then tear down their environment, we burn around 10 mins of computation credits per run. So I guess 2 containers are sufficient for the moment...